### ISSUE TITLE
How important is it to support a RH-like distribution?

### ISSUE THREAD

**Issue by mikalstill** [shakenfist/shakenfist#1173]
What portion of the market do we exclude by only doing Ubuntu and Debian? Should we support CentOS, Rocky, or RHEL? Do those distros have modern enough components to work for us?

**Comment by dth99** [comment]
yes, CentOs or RHEL are beyond good. If we talk about components and speed/performance there is no competition of RedHat based Operating Systems in Market.

**Comment by mikalstill** [comment]
Closing as out of scope. Supporting RHEL/Rocky/CentOS conflicts with the project design decision to target Ubuntu/Debian only (see CLAUDE.md, "Key Design Decisions": Ubuntu/Debian only, for maintainability). Reopen if that direction changes.

*(Triaged with Claude Code; details in CATEGORIZED_BUGS.md on the `bug-cleanup` branch.)*
